### PR TITLE
large frontend additions

### PR DESCRIPTION
**Pull request by mahirs0309**

### CODE REVIEW

File: bazel-testlogs [hunk 1]
```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_mahir/3a50be9f9914f6246922ce5499671600/execroot/arena/bazel-out/darwin_arm64-fastbuild/testlogs
```
Comment (ianmkim) [inline bazel-testlogs:1]: these bazel files should not be committed, please add them to gitignore

File: frontend/arena_frontend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Learn more https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files
```
Comment (ianmkim) [inline frontend/arena_frontend/.gitignore:1]: gitignore should be at the root of the repo, not just in the frontend

File: frontend/arena_frontend/screens/PendingBetsScreen.js
Comment (ianmkim) [inline frontend/arena_frontend/screens/PendingBetsScreen.js:1]: Empty file?

**Review by ianmkim — CHANGES_REQUESTED** [review]

File: frontend/arena_frontend/screens/PendingBetsScreen.js
Comment (mahirs0309) [reply frontend/arena_frontend/screens/PendingBetsScreen.js:1]: yeah its chill will fill it later

Comment (mahirs0309) [reply frontend/arena_frontend/screens/PendingBetsScreen.js:1]: unless u want me to delete it

Comment (ianmkim) [reply frontend/arena_frontend/screens/PendingBetsScreen.js:1]: Nah all good

**Review by ianmkim — APPROVED** [review]

**Review by ianmkim — APPROVED** [review]

**Pull request merged**